react-time-track/src/components/input-info-project.js

```diff
@@ -0,0 +1,157 @@
+/**@jsx jsx */
+import React from "react";
+import { jsx } from "@emotion/core";
+import { navigate } from "@reach/router";
+
+import { Button } from "./ui";
+
+function InputInfoProject() {
+  const today = new Date()
```

Comment:
It is required to set in stone certain requirements. About how we'll manage dates for example.

Reply:
I partially agree with you. Maybe, in some cases with more complexity where you need to work with dates a lot of. In this case, I didn't see that necessity.